ops/test/api.tf

```diff
@@ -39,6 +39,7 @@ module "simple_report_api" {
     DYNAMICS_CLIENT_SECRET                = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.dynamics_client_secret.id})"
     DYNAMICS_TENANT_ID                    = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.dynamics_tenant_id.id})"
     DYNAMICS_RESOURCE_URL                 = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.dynamics_resource_url.id})"
+    CURL_TIMEOUT                          = "4"
```

Comment:
@benwarfield-usds is this what you meant when you said that we should bump up the duration of the readiness check in `test`, or is there a more appropriate mechanism to do so?

Reply:
No, it needs to be set in the env for that step of the deployment workflow (or the whole workflow, for that matter, but the short timeout is useful for consistency on the `wait-for-%` targets).

Reply:
Ok, thanks. Removing!